Lib/fontbakery/checkrunner.py

```diff
@@ -1439,6 +1441,9 @@ def auto_register(self, symbol_table, filter_func=None):
       if filter_func returns a falsy value for an item, the item will
       not be registered.
     """
+    if spec_imports:
+      symbol_table['spec_imports'] = spec_imports
```

Comment:
This changes `symbol_table`, I'm not sure if that this is a good idea. Make a "defensive copy" first?


```py
symbol_table = symbol_table.copy()
symbol_table['spec_imports'] = spec_imports
```